SU-HSDO/suhumsci
# BUGHERD Tweaks

# READY FOR REVIEW

# Summary
- Changed border color of search to match comp (and Kerri's latest design file template)
- Reduced font-weight and increase font-size for sidebar navigation (should match main nav)
- Added active menu styles for main and subnav (plus Archaeology-specific state styles)

# Needed By (Date)
- End of sprint (7/19)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Check that the above outlined changes are propagating

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-152
- https://stanfordits.atlassian.net/browse/ARCH-169
- https://stanfordits.atlassian.net/browse/ARCH-170

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
